### PR TITLE
Initial Rails scaffold

### PR DESCRIPTION
**Pull request by TanyaPushchinska**

### CODE REVIEW

**Comment by vyanishevskyi** [comment]
In general I would advises for future:
1. As we have told yesterday - split you work into modular self-explained commits. For this PR would be much better to put rubocop corrections into new commit, since it is difficult to find what has been added related to the Animal entity.
2. Naming `Initial Railsscaffold` is not that descriptive and clear for that commit.
Would be better to call first commit something like: **Generate scaffold for Animal entity**
And create another one for rubocop correction calling it: **Autocorrect project with Rubocop**
3. That staff for rubocop could be even moved to another Pull Request, since it is better to have Pull Requests for single feature/improvement/bug, etc.
@SidiromUA @railkun @vdiakiv

File: db/migrate/20190701102552_create_animals.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateAnimals < ActiveRecord::Migration[5.2]
+  def change
+    create_table :animals do |t|
+      t.string :title
+      t.string :name
+      t.text :description
+
```
Comment (vyanishevskyi) [inline db/migrate/20190701102552_create_animals.rb]: What is the difference between `title` and `name` for animal here? Could animal have `title` at all? 🤔

File: config/database.yml [hunk 2]
```diff
@@ -15,11 +15,11 @@
...
 #
 default: &default
+  host: localhost
   adapter: postgresql
-  encoding: unicode
-  # For details on connection pooling, see Rails configuration guide
-  # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: tanya
+  password: tanya
```
[the remark this answers is not in the document]
Comment (TanyaPushchinska) [reply config/database.yml]: I've already deleted my personal data and will now move this file to .gitignore)

File: db/migrate/20190701102552_create_animals.rb (hunk 1 above)
Comment (TanyaPushchinska) [reply db/migrate/20190701102552_create_animals.rb]: Well, I understood this mistake))

**Pull request merged**